client/task-list/index.js

```diff
@@ -33,6 +33,33 @@ export class TaskDashboard extends Component {
 		document.body.classList.add( 'woocommerce-task-dashboard__body' );
 	}
 
+	trackStartedTask = ( taskName ) => {
+		const { userPreferences } = this.props;
+		const trackedStartedTasks =
+			userPreferences.task_list_tracked_started_tasks || {};
+		const currentTaskCount =
+			trackedStartedTasks && trackedStartedTasks[ taskName ]
+				? trackedStartedTasks[ taskName ]
+				: 0;
+		// Only record task visits up to 2 times
+		if ( currentTaskCount > 1 ) {
```

Comment:
Why not continue to count the task visits? This could be useful data - especially if added to the [`task_view` event](https://github.com/woocommerce/woocommerce-admin/blob/542cae22a5a4d2abdbfa2fcc2db8906a021cecfb/client%2Ftask-list%2Ftasks.js#L33).

Reply:
Added it to the `tasklist_click` event in the end, as that is being called on any of the tasks. Not sure if it's a bug that `task_view` event is only being triggered for the payment task?

Reply:
> Not sure if it's a bug that `task_view` event is only being triggered for the payment task?

Are you sure? Looking at the `<TaskList>` it seems that the event should fire for all tasks: https://github.com/woocommerce/woocommerce-admin/blob/6dddef0cbcdd05b6e93347bfc87a830c08133928/client%2Ftask-list%2Flist.js#L50

Reply:
I see both events fired:
![Screen Shot 2020-12-14 at 2 20 40 PM](https://user-images.githubusercontent.com/63922/102137196-7061be80-3e28-11eb-83a6-b52d730cb891.png)

Reply:
Ooh nice, I didn't realize the `recordTaskViewEvent` was exported, and only saw it used once in the file it was defined.
This makes more sense.